rpm/update-spec-provides.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# This script will update the goimports in the rpm spec for downstream fedora
+# packaging, via the `propose-downstream` packit action.
+# The goimports don't need to be present upstream.
+
+set -e
+
+PACKAGE=$(basename $(git rev-parse --show-toplevel))
+# script is run from git root directory
+SPEC_FILE=rpm/$PACKAGE.spec
+
+sed -i '/Provides: bundled(golang.*/d' $SPEC_FILE
+
+GO_IMPORTS=$(golist --imported --package-path github.com/containers/$PACKAGE --skip-self | sort -u | xargs -I{} echo "Provides: bundled(golang({}))")
```

Comment:
I'm not sure in this case, but might confirm that `-I{}` doesn't need to be quoted to protect it from bash gobbling it up.